### PR TITLE
Avoid error when known_hosts is not created or there are no ssh keys

### PR DESCRIPTION
**Pull request by jsarola**
Adding test for existing file known_host and ssh key created before container created

### CODE REVIEW

File: create-container.sh [hunk 1]
```diff
@@ -106,6 +106,16 @@ if [ ! -v BASE_PATH ] ; then
...
 fi
 
+# Test if known_hosts file exists
+if [ ! -f ~/.ssh/know_hosts ] ; then
+  touch ~/.ssh/know_hosts
+fi
+
+# Test if public key created
+if [ ! -f ~/.ssh/id_rsa.pub ] ; then
+  ssh-keygen -t rsa -N "" -q -f "$HOME/.ssh/id_rsa"
```
Comment (danypr92) [inline create-container.sh]:
```suggestion
  echo "I can't find a SSH public key in the default path: `$HOME/.ssh/id_rsa`."
  echo "You can use the var SSH_KEY_PATH to set a different path if you don't use the default."
  exit 0
```

**Review by danypr92 — COMMENTED** [review]
Thanks, @jsarola, for the contribution and sorry for the long time to response!

These two tests are perfect, we need to be sure that the folders and fields that we need to use exists, but I think that create the SSH key is too much. We can raise an error with a message with the instructions to use the var SSH_KEY_PATH if you don't use the default path.

What do you think?

File: create-container.sh [hunk 2]
```diff
@@ -106,6 +106,16 @@ if [ ! -v BASE_PATH ] ; then
   BASE_PATH="/opt"
 fi
 
+# Test if known_hosts file exists
+if [ ! -f ~/.ssh/know_hosts ] ; then
+  touch ~/.ssh/know_hosts
+fi
+
+# Test if public key created
+if [ ! -f ~/.ssh/id_rsa.pub ] ; then
```
Comment (danypr92) [inline create-container.sh:115]: Use the `SSH_KEY_PATH` var in this condition.

Comment (jsarola) [reply create-container.sh:115]: I liked this way!

File: create-container.sh (hunk 1 above)
Comment (jsarola) [reply create-container.sh]: I liked this way!

File: create-container.sh [hunk 3]
```diff
@@ -106,6 +106,16 @@ if [ ! -v BASE_PATH ] ; then
   BASE_PATH="/opt"
 fi
 
+# Test if known_hosts file exists
+if [ ! -f ~/.ssh/know_hosts ] ; then
```
Comment (enricostano) [inline create-container.sh]:
```suggestion
if [ ! -f ~/.ssh/known_hosts ] ; then
```

File: create-container.sh [hunk 4]
```diff
@@ -106,6 +106,16 @@ if [ ! -v BASE_PATH ] ; then
...
+  touch ~/.ssh/know_hosts
```
Comment (enricostano) [inline create-container.sh]:
```suggestion
  touch ~/.ssh/known_hosts
```

**Comment by enricostano** [comment]
@jsarola would you mind to accept code suggestions? Thanks!

**Comment by jsarola** [comment]
Code suggestions  accepted.

**Comment by enricostano** [comment]
@jsarola hi! would you mind to accept all the suggestions? Thanks! (reopening...)

File: create-container.sh (hunk 2 above)
Comment (enricostano) [reply create-container.sh:115]: @jsarola this change disappered, right?

File: create-container.sh (hunk 1 above)
Comment (enricostano) [reply create-container.sh]: @jsarola this change also disappered, right?

**Comment by enricostano** [comment]
@jsarola let's keep this PR open, please just add the changes that disappeared so we can merge it.

**Comment by jsarola** [comment]
I push the new changes.

**Comment by enricostano** [comment]
Many thanks @jsarola @somenergia-it !

**Pull request merged**